starter-core/src/main/java/io/micronaut/starter/application/Project.java

```diff
@@ -20,26 +20,30 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class Project {
+public class Project extends ProjectIdentifier {
 
-    private final String packageName;
+    public static final String PACKAGE_NAME = "packageName";
+    public static final String PACKAGE_PATH = "packagePath";
+    public static final String CLASS_NAME = "className";
+    public static final String NATURAL_NAME = "naturalName";
+    public static final String PROPERTY_NAME = "propertyName";
+    public static final String NAME = "name";
```

Comment:
I think this can be `private` or at least package protected.

Reply:
I changed them to `private`.